target/linux/realtek/dts/rtl8380_netgear_gigabit.dtsi

```diff
@@ -0,0 +1,141 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "rtl838x.dtsi"
+
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	compatible = "realtek,rtl838x-soc";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x00000000 0x08000000>;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
+
+		mode {
+			label = "reset";
+			gpios = <&gpio0 24 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+};
+
+&gpio0 {
+	indirect-access-bus-id = <0>;
+};
+
+&spi0 {
+	status = "okay";
+
+	flash@0 {
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <50000000>;
+
+		partitions {
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "u-boot";
+				reg = <0x00000000 0x000e0000>;
```

Comment:
Would have been easier to read without the useless additional leading zero added here (compared to before copied).

Reply:
I originally had this without the leading zeros, but someone asked me to add them in a previous diff review.  Should I keep them or remove them?

Reply:
It's a good idea to have the same number of digits for all partitions, but 7 would be enough here, and adding the eighth number just makes it harder to read again.

Reply:
Sounds good.  I've trimmed the leading zeros (to the highest significant hex digit) for both the flash offset (7)  and size (6).

Reply:
But that makes comparing offset and size quite hard actually. Could you please just use 7 digits for all values like it was before you changed the copied source, just like I asked for initially?

Reply:
Sure.